### ISSUE TITLE
Let user know if `Perf` is not available on global scope

### ISSUE THREAD

**Issue by danielberndt** [crysislinux/chrome-react-perf#7]
I know, it's all in the readme, but right now users will be quite confused when they fail to setup their app properly since there's no (error) message telling them what went wrong.

**Comment by crysislinux** [comment]
Yeah, that's a problem.  I plan to make it similar to redux devtools extension, show error message instead if window.Perf is not exist.
